sdks/java/io/mongodb/src/main/java/org/apache/beam/sdk/io/mongodb/MongoDbGridFSIO.java

```diff
@@ -174,6 +174,7 @@ public void parse(GridFSDBFile input, ParserCallback<String> callback)
         .setParser(TEXT_PARSER)
         .setCoder(StringUtf8Coder.of())
         .setConnectionConfiguration(ConnectionConfiguration.create())
+        .setSkew(Duration.ZERO)
```

Comment:
Yes. `getAllowedTimestampSkew` is not permitted to return `null` but if unset `MongoDbGridFSIO` was doing so. Obtaining the value of `getMillis` from the returned skew was throwing a `NullPointerException` when running these tests.